gemfiles/code_ownership.gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+gem "code_ownership"
```

Comment:
Shall we pin it to a specific version? Not a rhetorical question, I don't know what the other gemfiles do.